Microsoft.Toolkit.Uwp/Helpers/ResourceHelper.cs

```diff
@@ -0,0 +1,35 @@
+using System.Diagnostics;
+using Windows.ApplicationModel.Resources.Core;
+
+namespace Microsoft.Toolkit.Uwp
+{
+    public static class ResourceHelper
+    {
+        /// <summary>
+        /// Resource Helper to help get strings from localized Resourcefiles
+        /// </summary>
+        /// <param name="resourceString">Put in string Resource like this --> "nameOfTheResourceString"</param>
+        /// <returns>Output is a string with the value of the resource string the user asked for</returns>
+        public static string GetString(string resourceString)
+        {
+            // Connect to the resource-files
+            ResourceContext defaultContext = ResourceContext.GetForCurrentView();
+            ResourceMap srm = ResourceManager.Current.MainResourceMap.GetSubtree("Resources");
```

Comment:
I'm not sure, but I think the file may not always be called "Resources". There is also ResourceLoader class, that can be used to retrieve resources, but I'm not sure which one is better to use.
`ResourceLoader.GetForViewIndependentUse()`